config.go

```diff
@@ -1,14 +1,43 @@
 package main
 
-import "fmt"
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+type networkEndpointGroupConfiguration struct {
+	Name string `json:"name"`
+}
+
+type backendServiceConfiguration struct {
+	Name     string `json:"name"`
+	CDN      bool   `json:"cdn"`
+	Affinity string `json:"affinity"`
+	Host     string `json:"host"`
+	Path     string `json:"path"`
+}
 
 type healthCheckConfiguration struct {
+	Name string `json:"name"`
 	Type string `json:"type"`
 	Path string `json:"path"`
 }
 
+type mode string
+
+const (
+	reuseMode   = "REUSE"
+	tagMode     = "TAG"
+	defaultMode = "DEFAULT"
```

Comment:
By convention, constant names are usually written in uppercase letters. This is for their easy identification and differentiation from variables in the source code.